app/src/importlayersdialog.cpp

```diff
@@ -45,19 +45,24 @@ void ImportLayersDialog::getFileName()
     FileDialog fd(this);
     mFileName = QFileDialog::getOpenFileName(this, tr("Choose file"),
                                              fd.getLastOpenPath(FileType::ANIMATION),
-                                             tr("Pencil Animation file (*.pclx)"));
+                                             tr("Pencil Animation file (*.pclx *.pcl)"));
```

Comment:
Is it possible to use `PFF_PROJECT_EXT_FILTER` instead?

Reply:
The function I need from FileDialog is openFileFilters(), and it is private, so I don't know how to retrieve it. Is there another way?

Reply:
You can get PFF_PROJECT_EXT_FILTER from fileformat.h

Reply:
Ok. Now I've got it. Include fileformat.h, and use PFF_PROJECT_EXT_FILTER.
Done